data/brands/tourism/hotel.json

```diff
@@ -1734,6 +1734,16 @@
         "tourism": "hotel"
       }
     },
+    {
+      "displayName": "Paradores",
+      "locationSet": {"include": ["es", "pt"]},
+      "note": "No name for Paradores's brand as each hotel is named differently.",
+      "tags": {
+        "brand": "Paradores de Turismo de España",
```

Comment:
Based on the `displayName`, I take it that “Paradores” is the most widely recognized name and “Paradores de Turismo de España” is an official name. If that’s the case, then `brand` can be just “Paradores”. We can add a `matchName` of `paradores de truism de españa` in case someone searches for the full name.

Reply:
Ok, I'll update it accordingly

Reply:
Done